codecov.yml

```diff
@@ -1,3 +1,9 @@
+codecov:
+  require_ci_to_pass: yes
+  notify:
+    after_n_builds: 24
+    wait_for_ci: yes
```

Comment:
this didn't work as expected. But at least codecov will update the status multiple times as more coverages are being reported